docs/conda.rst

```diff
@@ -0,0 +1,275 @@
+itself, are listed for installation. In addition to Horovod we typically will also use :code:`pip` 
+to install JupyterLab extensions to enable GPU and CPU resource monitoring via 
+`jupyterlab-nvdashboard`_ and Tensorboard support via `jupyter-tensorboard`_. ::
+
+    horovod==0.19.*
+    jupyterlab-nvdashboard==0.2.*
+    jupyter-tensorboard==0.2.*
+
+    # make sure horovod is re-compiled if environment is re-built
+    --no-binary=horovod
+
+Note the use of the :code:`--no-binary` option at the end of the file. Including this option 
+insures that Horovod will be re-built whenever the Conda environment is re-built.
+
+Building Conda environment
+--------------------------
+
+After adding any necessary dependencies that should be downloaded via Conda to the 
+:code:`environment.yml` file and any dependencies that should be downloaded via :code:`pip` to the 
+:code:`requirements.txt` file you create the Conda environment in a sub-directory :code:`env` of 
+your project directory by running the following commands.
+
+.. code-block:: bash
+
+    $ export ENV_PREFIX=$PWD/env
+    $ export HOROVOD_CUDA_HOME=$CUDA_HOME
+    $ export HOROVOD_NCCL_HOME=$ENV_PREFIX
+    $ export HOROVOD_GPU_OPERATIONS=NCCL
+    $ conda env create --prefix $ENV_PREFIX --file environment.yml --force
+
+By default Horovod will try and build extensions for all detected frameworks. See the 
+documentation on `environment variables`_ for the details on additional environment variables that 
+can be set prior to building Horovod.
+
+Once the new environment has been created you can activate the environment with the following 
+command.
+
+.. code-block:: bash
+
+    $ conda activate $ENV_PREFIX
+
+The :code:`postBuild` file
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you wish to use any JupyterLab extensions included in the :code:`environment.yml` and 
+:code:`requirements.txt` files, then you may need to rebuild the JupyterLab application.
+
+For simplicity, we typically include the instructions for re-building JupyterLab in a 
+:code:`postBuild` script. Here is what this script looks like for my Horovod environments.
+
+.. code-block:: bash
+
+    jupyter labextension install --no-build jupyterlab-nvdashboard 
+    jupyter labextension install --no-build jupyterlab_tensorboard
+    jupyter lab build
+
+Use the following commands to source the :code:`postBuild` script.
+
+.. code-block:: bash
+
+    $ conda activate $ENV_PREFIX # optional if environment already active
+    $ . postBuild
+
+Listing the contents of the Conda environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To see the full list of packages installed into the environment run the following command.
+
+.. code-block:: bash
+
+    $ conda activate $ENV_PREFIX # optional if environment already active
+    $ conda list
+
+Verifying the Conda environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+After building the Conda environment you can check that Horovod has been built with support for 
```

Comment:
environment you can check --> environment, check

Reply:
Done